src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -4815,8 +4815,8 @@ def _put_managed_cluster_ensuring_permission(
     return cluster
 
 
-def _ensure_cluster_identity_permission_on_kubelet_identity(cli_ctx, cluster_identity_object_id, scope):
-    factory = get_auth_management_client(cli_ctx, scope)
+def _ensure_cluster_identity_permission_on_kubelet_identity(cmd, cluster_identity_object_id, scope):
```

Comment:
I would suggest don't pass the big cmd in.

Reply:
Sure, will open another to replace cmd.